### PR TITLE
spdx: add GPL-3.0-or-later license

### PR DESCRIPTION
**Pull request by prabhu**
GPL-3.0-or-later license

Thanks for helping us make a better snapd!
Have you signed the [license agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution guide](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: spdx/licenses.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ var osi = []string{
 	"AGPL-3.0",
 	"GPL-2.0",
 	"GPL-3.0",
+	"GPL-3.0-or-later",
```
Comment (bboozzoo) [inline spdx/licenses.go:65]: We should probably add a bunch of other identifiers that added in the latest SPDX license list, but I can take care of this once we merge your change.

**Comment by bboozzoo** [comment]
The CLA check is doing silly things again. Can you merge master and push?

**Comment by prabhu** [comment]
Tried merging now. I do have an account at launchpad.net. Is there any other account I should create?

Also agree with your comment on adding all the new identifiers. Happy to close this and send a separate PR for that?

**Comment by bboozzoo** [comment]
```
🛇 prabhu@shiftleft.io has no Launchpad account
```
You probably should the same email as one you used for your LP account.

**Comment by prabhu** [comment]
Yes, it is using the same email. I assume this is the Ubuntu One site?

**Comment by bboozzoo** [comment]
You need a launchpad account and go through this process: https://ubuntu.com/legal/contributors

**Comment by prabhu** [comment]
https://launchpad.net/~prabhusl

**Comment by bboozzoo** [comment]
@prabhu can you merge master and push again?

**Comment by bboozzoo** [comment]
Merged master myself and pushed already.

**Comment by bboozzoo** [comment]
@prabhu looks like the CLA has not been signed yet. Are you sure you used the same email address when committing the code?

**Comment by pedronis** [comment]
this is superseded by #9629 that will finally land soon

**Pull request closed without merging**